docs/suspensive.org/src/pages/en/docs/react-query/QueryClientConsumer.mdx

```diff
@@ -0,0 +1,128 @@
+import { Callout } from '@/components'
+
+# QueryClientConsumer
+
+<Callout type='experimental'>
+
+`<QueryClientConsumer/>` is an experimental feature, meaning this interface is subject to change.
+
+</Callout>
+
+This component allows the use of `useQueryClient` within JSX while bypassing React hook rules. It facilitates declarative access to the QueryClient in legacy class components or specific environments (such as SSR and micro-frontends).
```

Comment:
```suggestion
This component allows the use of `useQueryClient` within JSX while bypassing React hook rules.
```

I want to know SSR, micro-frontend case

Reply:
The reason for mentioning SSR and micro-frontend environments is to provide context around the availability of this component. 

In particular, I wanted to point out that SSR and micro-frontend environments each need to be accessed at client runtime and can be handled declaratively, but on second thought, I think this could be confusing for non-specific use cases, so I'm removing it.